### ISSUE TITLE
thread watcher colour

### ISSUE THREAD

**Issue by ansamech** [nebukazar/OneeChan#55]
hey, i was trying to change the colour of the current page in the thread watcher, ie. the one that is bold because thats the page you are on. i couldnt figure out the CSS to do it, couldnt find the proper selector name to use. do you know it? also, would be cool if you added that to the theme page, to customize.

thanks

**Comment by nebukazar** [comment]
You should be able to do it with
```
.current .watcher-unread,
.current .watcher-title, 
.current .fa.fa-times {
   color: #FFF !important;
}
```
For reference pressing F12 will bring up the developer tools in your browser.

**Comment by ansamech** [comment]
yep used f12 and DOMinspector and everything spent a good hour trying figure it out haha, thanks so much, worked perfectly!
